nix/postgresql/generic.nix

```diff
@@ -7,6 +7,8 @@ let
       , glibc, zlib, readline, openssl, icu, lz4, zstd, systemd, libossp_uuid
       , pkg-config, libxml2, tzdata, libkrb5, substituteAll, darwin
       , linux-pam
+      #orioledb specific
+      , perl, bison, flex, docbook_xsl, docbook_xml_dtd_45, docbook_xsl_ns, libxslt
```

Comment:
Would this bring them in for all (including non-oriole) builds and resulting flakes?

Reply:
no, at the opening they are just listed as function arguments/attributes to make available at eval and build time

Then here they are conditional only if oriole is built

https://github.com/supabase/postgres/blob/sam/oriole17/nix/postgresql/generic.nix#L87